addons/mail/static/src/utils/test_utils.js

```diff
@@ -895,4 +948,5 @@ export {
     nextTick,
     pasteFiles,
     start,
+    TEST_USER_IDS,
```

Comment:
If it works: export it where it's defined instead, default export object is a legacy way of doing things.